### DISCUSSION TITLE
Disable the "scroll on anchor" behavior

### DISCUSSION THREAD

**Discussion by asmeurer** [pradyunsg/furo#384] [Questions] [answered]
Is it possible to disable the behavior where going to an anchor scrolls the page? This is somewhat slow on larger pages, compared to just jumping to the anchor.

**Comment by pradyunsg** [comment]
Do have a specific page this is happening on? I'm pretty sure that I fixed this, like, a long time ago.

**Comment by asmeurer** [comment]
Try one of the longer pages here https://bertiewooster.github.io/. For example going to the bottom most item on https://bertiewooster.github.io/sympy-doc/furo/modules/matrices/common.html#sympy.matrices.common.MatrixKind takes a second. Compare that to, for instance, https://bertiewooster.github.io/sympy-doc/rtd/modules/matrices/common.html#sympy.matrices.common.MatrixKind, which jumps to the section instantly as soon at the page loads. I've also noticed this scrolling around behavior makes the back button laggy if you've been clicking around on the right sidebar, because it wants to scroll to each header rather than jumping to it. For example, try clicking random headers on https://bertiewooster.github.io/sympy-doc/furo/modules/functions/elementary.html and then clicking the back button a bunch. 

(and yes, I know we really need to split up our documentation so the pages aren't so huge, but there are some reasons that's not so straightforward for us to do)

**Comment by pradyunsg** [comment]
Given that https://github.com/sympy/sympy/pull/23159 already has a custom.css, you should be able to disable this by adding the following to it:

```
html {
  scroll-behavior: auto;
}
```

**Comment by asmeurer** [comment, marked as answer]
That seems to work. Didn't realize this could be done with CSS.
